### PR TITLE
ACI_STATIC_BINDING: Update paths paramater

### PR DESCRIPTION
**Pull request by jmcgill298**
##### SUMMARY

Change `paths` to `leafs` as the expected value is a leaf or pair of leafs(leaves).

Change the input to use a list of one or two leafs and validate the correct amount of entries are used based on whether it is a form of port-channel or a form of physical interface. An entry of one is acceptable as string.

Change representation to use 101 and 102 as those are the most common examples in ACI documentation.

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

aci_static_binding_to_epg
##### ANSIBLE VERSION

```
2.4
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by dagwieers — APPROVED** [review]
I made a small change, and now Shippable validates it correctly. LGTM

**Comment by nitzmahone** [comment]
@dagwieers This doesn't cherry-pick cleanly into stable-2.5 due to numerous overlapping changesets and conflicts; please file a PR against stable-2.5 with the conflicts adjusted and we'll get it in before rc1.

**Comment by dagwieers** [comment]
@nitzmahone That's weird, I would be expecting that if everything is applied in the same order it should merge without a problem (since all the ACI changesets only affect ACI and nobody else is doing ACI changes). But no worries, on Tuesday I will compare stable-2.5 and devel branches and make a PR of anything that fell off ;-)

**Comment by nitzmahone** [comment]
Yeah, it's the "applied in the same order" that's tricky- I was just working through the 2.5.0 blocker queue, so the order that the various PRs land in stable-2.5 is not guaranteed...

**Comment by nitzmahone** [comment]
I think we're about to change the cherry-pick procedures up a bit for the rest of 2.5 and 2.6- sounds like we're going to ask owners to do cherry-picks as PRs against the stable branches, so if you wanted to combine a bunch of things like this into a single PR in the future, that'd be legit... It also ensures both that nothing gets lost because the release manager forgot to pick it, and that the release manager won't potentially bork up a conflict resolution on code he doesn't own/know. It also ensure that tests are passing before they get merged, which we're pretty lax about on current cherry-picks (usually just direct push and watch the post-merge build on the branch). Basically just makes the release manager a reviewer and gatekeeper...

**Pull request merged**
